### ISSUE TITLE
Unspecified error [a20459352f]

### ISSUE THREAD

**Issue by aem-intellij-plugin-error-reporter** [aem-intellij-plugin-error-reporter/auto-generated-issues-storage#62]
fixed:0.11.3

## Description
*No description*
## Exception
```
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'file' of com/intellij/openapi/roots/impl/RootIndex.ensureValid must not be null
	at com.intellij.openapi.roots.impl.RootIndex.$$$reportNull$$$0(RootIndex.java)
	at com.intellij.openapi.roots.impl.RootIndex.ensureValid(RootIndex.java)
	at com.intellij.openapi.roots.impl.RootIndex.buildRootInfo(RootIndex.java:251)
	at com.intellij.openapi.roots.impl.RootIndex.<init>(RootIndex.java:66)
	at com.intellij.openapi.roots.impl.DirectoryIndexImpl.getRootIndex(DirectoryIndexImpl.java:119)
	at com.intellij.openapi.roots.impl.DirectoryIndexImpl.getInfoForFile(DirectoryIndexImpl.java:132)
	at com.intellij.openapi.roots.impl.FileIndexBase.getInfoForFileOrDirectory(FileIndexBase.java:95)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.getInfoForFileOrDirectory(ProjectFileIndexImpl.java:25)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.isExcluded(ProjectFileIndexImpl.java:79)
	at co.nums.intellij.aem.eW.visitFileEx(Unknown Source)
	at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:295)
	at co.nums.intellij.aem.eY.a(Unknown Source)
	at co.nums.intellij.aem.eY.a(Unknown Source)
	at co.nums.intellij.aem.eY.a(Unknown Source)
	at co.nums.intellij.aem.eV.<init>(Unknown Source)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:88)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:66)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:48)
	at com.intellij.openapi.components.impl.ServiceManagerImpl$MyComponentAdapter.createAndInitialize(ServiceManagerImpl.java:243)
	at com.intellij.openapi.components.impl.ServiceManagerImpl$MyComponentAdapter.getComponentInstance(ServiceManagerImpl.java:227)
	at com.intellij.util.pico.DefaultPicoContainer.getLocalInstance(DefaultPicoContainer.java:246)
	at com.intellij.util.pico.DefaultPicoContainer.getComponentInstance(DefaultPicoContainer.java:213)
	at com.intellij.openapi.components.ServiceManager.doGetService(ServiceManager.java:75)
	at com.intellij.openapi.components.ServiceManager.getService(ServiceManager.java:49)
	at co.nums.intellij.aem.fa.a(Unknown Source)
	at co.nums.intellij.aem.fn.runActivity(Unknown Source)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivity(StartupManagerImpl.java:205)
	at com.intellij.ide.startup.impl.StartupManagerImpl.lambda$null$1(StartupManagerImpl.java:181)
	at com.intellij.openapi.project.DumbServiceImpl.doRun(DumbServiceImpl.java:389)
	at com.intellij.openapi.project.DumbServiceImpl.updateFinished(DumbServiceImpl.java:381)
	at com.intellij.openapi.application.TransactionGuardImpl.runSyncTransaction(TransactionGuardImpl.java:82)
	at com.intellij.openapi.application.TransactionGuardImpl.lambda$pollQueueLater$0(TransactionGuardImpl.java:67)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:433)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
## Error context
```
Last Action:       null
Plugin Version:    0.11.1
OS Name:           Mac OS X
Java Version:      11.0.4
Java VM Vendor:    JetBrains s.r.o
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IU-192.7142.36
App Version:       2019.2.4
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       HideAllWindows
Plugin Version:    0.8.5
OS Name:           Windows 8
Java Version:      1.8.0_51
Java VM Vendor:    Oracle Corporation
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IC-143.1184
App Version:       15.0.2
```

**Comment by karollewandowski** [comment]
To the last error reporter: please update plugin to the newest version. Error reported was bugged in 0.8.5 and you probably reported something different.

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       null
Plugin Version:    0.8.1
OS Name:           Linux
Java Version:      11.0.4
Java VM Vendor:    JetBrains s.r.o
Product Name:      PhpStorm
Full Product Name: PhpStorm
App Version Name:  PhpStorm
Is EAP:            false
App Build:         PS-193.5233.101
App Version:       2019.3
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       
Plugin Version:    0.8.5
OS Name:           Windows 8
Java Version:      1.8.0_51
Java VM Vendor:    Oracle Corporation
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IC-143.1184
App Version:       15.0.2
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       EditorPaste
Plugin Version:    0.8.5
OS Name:           Windows 7
Java Version:      1.8.0_51
Java VM Vendor:    Oracle Corporation
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IC-143.381
App Version:       15.0
```

**Comment by karollewandowski** [comment]
Thank you for reporting it.
Fixed in 0.11.3, please update the plugin.
